### DISCUSSION TITLE
Getting GridListItem or Tag to use `a` tag when an href is passed?

### DISCUSSION THREAD

**Discussion by schurchleycci** [adobe/react-spectrum#7230] [Q&A] [answered]
With a `ListBox`, if you pass an href to a `ListBoxItem` it gets rendered using the `a` tag. This doesn't seem to be the case, however, for a `GridListItem` or a `Tag` - they get rendered as a `div`. Is this expected behaviour, missing functionality, or am I just missing something?

**Comment by snowystinger** [comment, marked as answer]
This is expected. `GridListItem`'s can have other interactive children. An `a` tag cannot have interactive children, so a `GridListItem` can never render as an `a` tag. Tags were implemented as a GridList, so have the same restriction.

**Reply by schurchleycci** [reply]
Ah, that makes sense. Are there any accessibility concerns with `GridListItem`s and `Tag`s acting as links without being an `a` tag? E.g. should they have an ARIA role of "link" (though I realize that they don't have a role `prop` so I couldn't manually add it)?

**Reply by devongovett** [reply]
It's no different from something like `<a role="menuitem">`. The `role` attribute overrides the link role, so it would never be announced as a link.

**Reply by schurchleycci** [reply]
Gotcha - thanks @snowystinger and @devongovett!
